src/yarle.ts

```diff
@@ -32,25 +33,31 @@ export const parseStream = async (options: YarleOptions): Promise<void> => {
 
       return reject();
     };
-  
+
     xml.collect('tag');
     xml.collect('resource');
-    xml.on('startElement: en-export', (enExport: any) => {
-      totalNotes = Array.isArray(enExport.note) ? enExport.note.length : 1;
-    });
 
     xml.on('endElement: note', (note: any) => {
-      processNode(note);
-      ++noteNumber;
-      console.log(`Notes processed: ${noteNumber}`);
+      if (
+          options.skipWebClips &&
+          (
+            note['note-attributes']['source-application'] === 'webclipper.evernote' ||
+            note['note-attributes']['source'] === 'web.clip7'
+          )
+        ) {
+          ++skipped;
+          console.log(`Notes skipped: ${skipped}`);
+      } else {
+        processNode(note);
+        ++noteNumber;
+        console.log(`Notes processed: ${noteNumber}`);
+      }
     });
 
     xml.on('end', () => {
       const success = noteNumber - failed;
       console.log(
-        `Conversion finished: ${success} succeeded, ${
-          totalNotes - success
-        } failed.`,
+        `Conversion finished: ${success} succeeded, ${skipped} skipped, ${failed} failed.`,
```

Comment:
please keep logging the total number of notes as well.

Reply:
Done.